tutorials/intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,677 @@
+
+## Introduction
+Collection, visualization, and analysis of geographical or spatial data.
+
+## Data types
+
+- Vector data represent lat-long coordinates
+- Raster data comprises of pixels with associated values
+
+---
+
+
+- Points
+
+ ![Points](points.png)
+
+- Lines
+
+ ![Lines](line-vector.png)
+
+- Polygons
+
+ ![Polygons](polygon-vector.png)
+
+- Raster layers/bands
+
+ ![Raster](map-algebra.png)
+
+Image sources: https://gisgeography.com/spatial-data-types-vector-raster/
+
+---
+
+## Google Earth Engine platform
+
+[Introductory video](https://www.youtube.com/watch?v=gKGOeTFHnKY&feature=youtu.be, "Video")
+
+[Code editor](https://code.earthengine.google.com/)
+
+- Cloud-based platform for planetary scale geospatial analysis
+- Uses Google's computational resources to reduce processing time
+- Massive archive of remote sensing data 
+- 200+ public datasets 
+- over 4000 new images every day 
+- over 5 million images 
+- over 30 petabytes of data
+
+Source: Google Earth Engine user summit
+
+---
+
+![Code Editor (Source: developers.google.com)](ee-editor.png)
+
+---
+
+## Basic functions
+
+### Declaring variables
+```javascript
+var variableName = ee.ContainerType(value);
+```
+A container (in the form ee.variabletype) is used to wrap up a native javascript object type so that Google's server can recognize its structure and perform operations on it
+
+###  Centering map
+```javascript
+Map.setCenter(long, lat, zoomLevel);
+```
+>Zoom level varies from 0 (no zoom) to 20 (highest zoom level)
+
+### Displaying metadata
+```javascript
+print(variableName);
+```
+You cannot print more than 5000 elements at once.
+
+### Adding a layer to the map
+```javascript
+Map.addLayer(variableName);
+```
+
+---
+
+## Variable types in Earth Engine
+
+### Strings
+
+```javascript
+var str = ee.String('This is a string. Or is it? It is.');
+```
+### Numbers
+```javascript
+var num = ee.Number(5);
+```
+
+### Arrays
+```javascript
+var arr = ee.Array([[5, 2, 3], [-2, 7, 10], [6, 6, 9]]);
+```
+
+### Lists
+```javascript
+var lis = ee.List([5, 'five', 6, 'six']);
+```
+### Dictionaries
+```javascript
+var dict = ee.Dictionary({five: 5, six: 6});
+```
+
+### And the fun stuff
+- Geometries
+- Features
+- Feature collections
+- Images
+- Image collections
+
+---
+
+## Geometries – declaration and types
+
+### Points
+```javascript
+var poi = ee.Geometry.Point(0, 45);
+```
+### Multi points
+```javascript
+var multi = ee.Geometry.MultiPoint(0, 45, 5, 6, 70, -56);
+```
+### Line string
+```javascript
+var lineStr = ee.Geometry.LineString([[0, 45], [5, 6], [70, -56]]);
+```
+### Multi line string
+```javascript
+var mlineStr =
+ee.Geometry.MultiLineString([[[0, 45], [5, 6], [70, -56]], [[0, -45], [-5, -6], [-70, 56]]]);
+```
+### Linear Ring
```

Comment:
Nit: Consistent capitalization on titles here and throughout (Proper case - first letter upper, the rest lower unless they're proper nouns or acronyms)